deploy/olm-catalog/nightly/eclipse-che-preview-openshift/manifests/devworkspaceroutings.controller.devfile.io.CustomResourceDefinition.yaml

```diff
@@ -0,0 +1,3708 @@
+#
+#  Copyright (c) 2019-2021 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: devworkspace-controller
+    app.kubernetes.io/part-of: devworkspace-operator
+  name: devworkspaceroutings.controller.devfile.io
```

Comment:
I prefer to run discussions in threads, so starting it instead of commenting PR.

DevWorkspace Routing CRD must not be shipped by Che Operator and even by DevWorkspace Che Operator.
It may work somehow now, but it's supposed to fully fail later when Descoping Operators is done.

So, devworkspace routing CRD must be installed only by one operator, and it's DevWorkspace Operator who really owns that API. Others (like DWCO) just depend on it.

The only proper solution I see is depending with OLM dependencies https://github.com/eclipse-che/che-operator/pull/890 in a dedicated channel ATM.
If we need the same Che Operator deployment in both channels, DWCO should be no-op is DevWorkspaceRouting CRD is not available, I think.